source/daplink/cmsis-dap/DAP.h

```diff
@@ -118,9 +118,11 @@
 #define DAP_ID_VENDOR                   1U
 #define DAP_ID_PRODUCT                  2U
 #define DAP_ID_SER_NUM                  3U
-#define DAP_ID_FW_VER                   4U
+#define DAP_ID_CMSIS_DAP_VER            4U
+#define DAP_ID_FW_VER                   4U      // Deprecated alias of DAP_ID_CMSIS_DAP_VER for backwards compatibility.
 #define DAP_ID_DEVICE_VENDOR            5U
 #define DAP_ID_DEVICE_NAME              6U
+#define DAP_ID_PRODUCT_FW_VER           7U
```

Comment:
The DAP_INFO entry for 0x07 is reserved for TARGET_BOARD_NAME (and 0x08 for TARGET_BOARD_VENDOR).

Recommend this is pushed out to a different address to keep compatibility.